mods/ra/weapons/smallcaliber.yaml

```diff
@@ -32,67 +9,92 @@ FLAK-23-AA:
 	Warhead@1Dam: SpreadDamage
 		Spread: 213
 		Damage: 20
-		ValidTargets: Air, Ground, Water
+		ValidTargets: Air
 		Versus:
 			None: 40
 			Wood: 10
 			Light: 60
 			Heavy: 10
 			Concrete: 20
 		DamageTypes: Prone50Percent, TriggerProne, DefaultDeath
-	Warhead@2EffAir: CreateEffect
+	Warhead@2Eff: CreateEffect
 		Explosions: small_explosion_air
-		ValidImpactTypes: Air, AirHit
+
+ZSU-23:
+	Inherits: ^AACannon
+	Burst: 2
+	BurstDelay: 5
+	ReloadDelay: 0
+	Range: 10c0
+	Projectile: Bullet
+		Speed: 3c340
+	Warhead@1Dam: SpreadDamage
+		Damage: 12
+		Versus:
+			None: 30
+			Wood: 75
+			Light: 75
+			Heavy: 100
+			Concrete: 50
+
+FLAK-23-AA:
+	Inherits: ^AACannon
```

Comment:
Yes, because it doesn't regress anything and might prevent situations where no explosion is displayed at all due to a "wrong" impact type (below `airMargin` in `CreateEffectWarhead`, for example).